ui/components/multichain/notification-detail-address/notification-detail-address.tsx

```diff
@@ -11,11 +11,11 @@ import { shortenAddress } from '../../../helpers/utils/util';
 import { toChecksumHexAddress } from '../../../../shared/modules/hexstring-utils';
 
 export type NotificationDetailAddressProps = {
-  side: string;
+  side: string | null;
   address: string;
 };
 
-const SideText: FC<{ side: string }> = ({ side }) => (
+const SideText: FC<{ side: string | null }> = ({ side }) => (
```

Comment:
Nit - why would we bother rendering the content below if this is null?

Reply:
In fact looking at implementations, is `side` ever null? I think you can revert this file change.

Reply:
You are right. Fixed!